Makefile

```diff
@@ -136,21 +144,20 @@ validate-swagger-changes:
 ##################
 
 .PHONY: build docs
-build: lint unit-test build-minimal
+build: lint $(UNIT_TEST_TARGET) build-minimal
 
 build-all: build-in-docker
 
 build-in-docker:
-	skipper make build-image
+	skipper make -e BUILD_TYPE=$BUILD_TYPE build-image
```

Comment:
From CI:
```
10:15:14  skipper make -e BUILD_TYPE=UILD_TYPE build-image
```

Please add the new variable to the `skipper.yaml` file, then, no need to pass it explicitly in here. :)